IFS/STARTHERE_IFS_Healpix.ipynb

```diff
@@ -30,7 +30,9 @@
    },
    "outputs": [],
    "source": [
-    "# uncomment below to install the hackathon python environment\n",
+    "#1)uncomment below to install the hackathon python environment (only needed once). \n",
+    "#2)After installing, refresh the jupyther browser and select as kernel the newly installed one (click on top-right, look for python-HH-hackathon)\n",
```

Comment:
Can you correct the spelling of Jupyter?

Reply:
Done :)